src/libm-tester/tester.c

```diff
@@ -29,7 +29,7 @@
 #include "testerutil.h"
 
 void stop(char *mes) {
-  fprintf(stderr, "%s\n", mes);
+  printf("%s\n", mes);
```

Comment:
Do you have any problem if I change the output of the tester from stderr to stdout?
For me, this is more convenient when I run many testers in parallel. There are many combinations of OS and compiler now, and how to run testers quickly with different configurations is becoming 
important. If this change is not desirable for your side, I will try to come up with another way.